src/modules/displaypopup.js

```diff
@@ -0,0 +1,39 @@
+import { createPopUpDetails, addComment } from './createpopup.js';
+import { generateComment } from './rendercomment.js';
+
+const showDetails = async (id) => {
+  const baseApi = 'https://api.tvmaze.com/shows/';
+  const list = await fetch(`${baseApi}${id}`).then((response) => response.json());
+  return list;
+};
+const addClosePopUpevent = () => {
+  const closeBtn = document.querySelector('.cancel');
+  closeBtn.addEventListener('click', () => {
+    const popupModal = document.querySelector('.window-popup');
+    popupModal.style.display = 'none';
+  });
+};
+
+const displayPoUp = async (id) => {
+  const popupModal = document.querySelector('.window-popup');
+
+  popupModal.innerHTML = '';
+  const movieDetail = await showDetails(id);
+  popupModal.append(createPopUpDetails(movieDetail));
+  popupModal.style.display = 'flex';
+  const commentLine = document.querySelector('.con-comm');
+  const comments = await generateComment(id);
+  commentLine.append(comments);
+  addClosePopUpevent();
+  addComment();
+};
+
+/* eslint-disable */
+export const addCommentPopupEvent = () => {
+  const commentBtns = document.querySelectorAll('.btn');
+  commentBtns.forEach((btn) => btn.addEventListener('click', (e) => {
+    e.preventDefault();
+    displayPoUp(btn.id);
+  }));
+};
+/* eslint-enable */
```

Comment:
- [ ] Kindly enable the linters here too. 🤓

Reply:
[Issue resolved](https://github.com/Wahaj-Ali/tv-hub/commit/eded83a40c421c398b65fc9fa4a85a3e616a6af8)